src/helper-modules/ordnance-survey/dto/get-search-postcode-ordnance-survey-query.dto.ts

```diff
@@ -0,0 +1,3 @@
+export class GetSearchPostcodeOrdnanceSurveyQueryDto {
+  public postcode: string;
+}
```

Comment:
I don't think that this class is used anywhere.

Reply:
you are right, removing it.